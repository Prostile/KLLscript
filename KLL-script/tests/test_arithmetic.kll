int a;
int b;
int c;
int result;

cin(a); // ¬вод: 10
cin(b); // ¬вод: 5
cin(c); // ¬вод: 3

result = a + b * c; // 10 + 5 * 3 = 25
cout(result);

result = (a + b) * c; // (10 + 5) * 3 = 45
cout(result);

result = a / b - c; // 10 / 5 - 3 = 2 - 3 = -1 (целочисленное деление)
cout(result);

result = a * (b - c + 1); // 10 * (5 - 3 + 1) = 10 * 3 = 30
cout(result);