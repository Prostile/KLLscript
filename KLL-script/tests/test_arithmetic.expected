25
45
-1
30